packages/kotti-banner/src/Banner.vue

```diff
@@ -65,7 +65,7 @@ export default {
 			}
 		},
 		expandable() {
-			return this.$slots.expand
+			return !!this.$slots.expand
```

Comment:
```suggestion
			return Boolean(this.$slots.expand)
```

Reply:
agreed. @carsoli can you commit this suggestion?

Reply:
i don't even know why this commit is here, it should be on #126 :D but i was going to add it to 126...